scripts/fix-shrinkwrap.js

```diff
@@ -1,46 +1,56 @@
+'use strict';
+
+const _ = require('lodash');
 const fs = require('fs');
 const path = require('path');
 
-var time = Date.now();
+// No magic numbers it is, eslint
+const ZERO = 0;
+const ONE = 1;
+const TWO = 2;
+const THOUSAND = 1000;
+
+let time = Date.now();
 
 // Load the shrinkwrap
 const SHRINKWRAP_FILE = path.join(__dirname, '..', 'npm-shrinkwrap.json');
 const shrinkwrap = JSON.parse(fs.readFileSync(SHRINKWRAP_FILE, 'utf8'));
 
-// Fix up an issue with the shrinkwrap caused by npm@5.0.3,
-// where nested dependencies in the shrinkwrap have the "resolved"
-// URL as "version" key, and are missing the "resolved" key
-function fixup(dependency) {
+/**
+ * Fix up an issue with the shrinkwrap caused by npm@5.0.3,
+ * where nested dependencies in the shrinkwrap have the "resolved"
+ * URL as "version" key, and are missing the "resolved" key
+ * @param {Object} dependency - dependency
+ * @returns {Boolean}
+ */
+const fixup = (dependency) => {
 
   // If the dependency version is a URL and
   // the "resolved" key is missing, we need to fix it
   if (!dependency.resolved && /^http(s):/i.test(dependency.version)) {
 
     // Extract the version number from the registry URL
-    var match = dependency.version.match(/(\d+\.\d+\.\d+(\-.+)?)\.tgz$/);
+    const match = dependency.version.match(/(\d+\.\d+\.\d+(-.+)?)\.tgz$/);
 
     // If something goes wrong, we do not want
     // this to continue in any way; so we crash
-    if (!match || !match[1]) {
+    if (!match || !match[ONE]) {
```

Comment:
Alternatively, you could change the regex to use a non-capturing group `(?: )`, and then use `_.first` ?